### PR TITLE
feat(helper-image): add option to not add exif data

### PR DESCRIPTION
**Pull request by aimeex3**

### CODE REVIEW

File: packages/node_modules/@webex/helper-image/src/index.js [hunk 1]
```diff
@@ -25,7 +27,13 @@ export function updateImageOrientation(file) {
       resolve(buf);
     };
   })
-    .then((buf) => readExifData(file, buf));
+    .then((buf) => {
+      if (options.shouldNotAddExifData) {
+        return buf;
+      }
+
+      return readExifData(file, buf);
```
Comment (adamweeks) [inline packages/node_modules/@webex/helper-image/src/index.js:35]: I like how `readExifData` is actually `addExifData` 🤦‍♂️

File: packages/node_modules/@webex/helper-image/src/index.js [hunk 2]
```diff
@@ -11,9 +11,11 @@ const {ExifImage} = require('exif');
 /**
 * Updates the image file with exif information, required to correctly rotate the image activity
 * @param {Object} file
+* @param {Object} options
+* @param {Object} options.shouldNotAddExifData
```
Comment (adamweeks) [inline packages/node_modules/@webex/helper-image/src/index.js:15]:
```suggestion
* @param {boolean} options.shouldNotAddExifData
```

File: packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js [hunk 3]
```diff
@@ -334,9 +334,10 @@ const Conversation = WebexPlugin.extend({
    * @param {string} item.url
    * @param {Object} options
    * @param {Object} options.headers
+   * @param {Object} options.shouldNotAddExifData
```
Comment (adamweeks) [inline packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js:337]:
```suggestion
   * @param {boolean} options.shouldNotAddExifData
```

**Comment by aimeex3** [comment]
@adamweeks thanks for reviewing. updated PR to address comments

**Pull request merged**